### PR TITLE
[INLONG-2912][Manager] Add fields for the binlog task

### LINKED ISSUE #2912: [Improve][Manager] Add fields for the binlog task
### Description

Add fields for the binlog task of the agent module.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Title Name: [INLONG-2912][Manager] Add fields for the binlog task

Fixes #2912 

### Motivation

Add fields for the binlog task.

### Verifying this change

- [X] Make sure that the change passes the CI checks.

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSourceTransfer.java [hunk 1]
```diff
@@ -160,23 +178,27 @@ private static KafkaSourceRequest createKafkaSourceRequest(KafkaSource kafkaSour
 
     private static BinlogSourceRequest createBinlogSourceRequest(MySQLBinlogSource binlogSource,
             InlongStreamInfo streamInfo) {
-        BinlogSourceRequest binlogSourceRequest = new BinlogSourceRequest();
-        binlogSourceRequest.setSourceName(binlogSource.getSourceName());
-        binlogSourceRequest.setInlongGroupId(streamInfo.getInlongGroupId());
-        binlogSourceRequest.setInlongStreamId(streamInfo.getInlongStreamId());
-        binlogSourceRequest.setSourceType(binlogSource.getSourceType().name());
+        BinlogSourceRequest sourceRequest = new BinlogSourceRequest();
```
Comment (EMsnap) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSourceTransfer.java:181]: Use builder might be better

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java [hunk 2]
```diff
@@ -176,13 +176,25 @@ private TaskResult getTaskResult(TaskRequest request) {
         List<DataConfig> dataConfigs = Lists.newArrayList();
         List<StreamSourceEntity> entityList = sourceMapper.selectByIpAndUuid(agentIp, uuid);
         for (StreamSourceEntity entity : entityList) {
+            // Change 20x to 30x
+            int id = entity.getId();
+            int status = entity.getStatus();
+            int op = status % 100;
```
Comment (EMsnap) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java:182]: magic num

**Review by EMsnap — COMMENTED** [review]
PLAL

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java]: Fixed.

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]
LGTM

**Pull request merged**